src/SqliteSampleBlock.cpp

```diff
@@ -782,13 +809,20 @@ void SqliteSampleBlock::Delete()
    // preconditions; should return SQL_OK which is 0
    if (sqlite3_bind_int64(stmt, 1, mBlockID))
    {
+      ADD_EXCEPTION_CONTEXT(
+         "sqlite3.rc", std::to_string(sqlite3_errcode(Conn()->DB())));
+      ADD_EXCEPTION_CONTEXT("sqlite3.context", "SqliteSampleBlock::Delete.bind");
+
       wxASSERT_MSG(false, wxT("Binding failed...bug!!!"));
    }
 
    // Execute the statement
    rc = sqlite3_step(stmt);
    if (rc != SQLITE_DONE)
    {
+      ADD_EXCEPTION_CONTEXT("sqlite3.rc", std::to_string(rc));
+      ADD_EXCEPTION_CONTEXT("sqlite3.context", "SqliteSampleBlock::Delete.step");
+
```

Comment:
This whole commit is also straightforward, but not obviously complete from reading diffs alone.